app/app.go

```diff
@@ -741,6 +746,9 @@ func New(
 		transferStack.(delayedackmodule.IBCMiddleware),
 	))
 
+	// register the denommetadata hooks
+	app.DenomMetadataKeeper.SetHooks(denommetadatamoduletypes.NewMultiRollappHooks(app.DenomMetadataKeeper.DenomMetadataHooks()))
```

Comment:
this is for other modules who want to register those hooks. so this line is redundant currently as no module want to register hooks (will be added later by @VictorTrustyDev )

Reply:
removed